pytests/test_window.py

```diff
@@ -4,7 +4,60 @@
 from bytewax.execution import run_main
 from bytewax.inputs import TestingBuilderInputConfig
 from bytewax.outputs import TestingOutputConfig
-from bytewax.window import EventClockConfig, HoppingWindowConfig, TumblingWindowConfig
+from bytewax.window import (
+    EventClockConfig,
+    HoppingWindowConfig,
+    TumblingWindowConfig,
+    SessionWindow,
+)
+
+
+class Dt(datetime):
+    def after(self, s):
+        return self + timedelta(seconds=s)
+
+
+def test_session_window():
+    start_at = Dt(2022, 1, 1, tzinfo=timezone.utc)
+
+    flow = Dataflow()
+
+    def gen():
```

Comment:
Could you re-phrase this?

```py
inp = [
    ("ALL", ...),
    ...
]

flow.input("inp", TestingInputConfig(inp))
```

because we gave up on trying to futz with a global mocked clock and use event time, we don't actually need the whole "generator builder" pattern. And I'll be getting rid of the `TestingBuilderInputConfig` in my other PR.